container/docker/docker.go

```diff
@@ -161,3 +178,21 @@ func parseDockerVersion(full_version_string string) ([]int, error) {
 	}
 	return version_array, nil
 }
+
+// TODO: switch to a semantic versioning library.
```

Comment:
Looks like Kubernetes is using [blang/semver](https://github.com/kubernetes/kubernetes/search?utf8=✓&q=semver+-path%3Avendor+-path%3Atest++-path%3Acluster&type=Code)